### PR TITLE
fix(argo-cd): Server cert-manager certificate secretName to match documentation

### PR DESCRIPTION
**Pull request by rlees85**
Set ArgoCD server certificate manager certificate secret name to match the documentation.
https://argo-cd.readthedocs.io/en/latest/operator-manual/tls/

Note on DCO:

If the DCO action in the integration test fails, one or more of your commits are not signed off. Please click on the *Details* link next to the DCO action for instructions on how to resolve this.

Checklist:

* [x] I have updated the chart version in `Chart.yaml` following Semantic Versioning.
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] I have followed the testing instructions in the [contributing guide](https://github.com/argoproj/argo-helm/blob/master/CONTRIBUTING.md).
* [x] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/tree/master/community#contributing-to-argo).
* [x] My build is green ([troubleshooting builds](https://argoproj.github.io/argo-cd/developer-guide/ci/)).

Changes are automatically published when merged to `master`. They are not published on branches.

### CODE REVIEW

**Comment by rlees85** [comment]
Since nobody has looked at it yet I decided to re-work it slightly to let the secret name be variable. The default now matches the ArgoCD documentation - so it will work with `LoadBalancer` type deployments but now the secret name can also be changed - if that suits some people who are using ingress.

**Comment by rlees85** [comment]
@AtzeDeVries Just tagging you in case if this gets merged it might affect what you do/dont need to specify in your Helm values.

(I found you here: https://github.com/argoproj/argo-helm/pull/755)

**Review by mkilchhofer — APPROVED** [review]
Thank you for this contribution. It definitively makes sense to match with whats in the docs. LGTM

File: charts/argo-cd/values.yaml [hunk 1]
```diff
@@ -500,6 +500,7 @@ server:
     domain: argocd.example.com
     issuer: {}
```
Comment (mkilchhofer) [inline charts/argo-cd/values.yaml:501]: While reviewing this I thought about adding this (It got removed in a very old PR https://github.com/argoproj/argo-helm/pull/129):
```suggestion
    issuer:
      kind: # ClusterIssuer
      name: # letsencrypt
```
WDYT?

Comment (rlees85) [reply charts/argo-cd/values.yaml]: It can't hurt, I tried templating it and it doesn't break anything either so I have added it.

**Review by mkilchhofer — APPROVED** [review]

**Pull request merged**